native/packages/react-native-bpk-component-carousel/stories.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Image, Text, StyleSheet } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+import BpkPaginationDots from 'react-native-bpk-component-pagination-dots';
+import {
+  spacingMd,
+  colorGray300,
+  colorGray500,
+} from 'bpk-tokens/tokens/base.react.native';
+import range from 'lodash/range';
+import CenterDecorator from '../../storybook/CenterDecorator';
+import BpkCarousel, { BpkCarouselItem } from './index';
+
+const STYLES = StyleSheet.create({
+  page: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  view: {
+    alignSelf: 'center',
+    width: 300,
+    maxHeight: 300,
+  },
+});
+
+const renderKittenImages = () =>
+  [
+    'https://placekitten.com/g/300/300',
+    'https://placekitten.com/300/300',
+    'https://placekitten.com/g/400/400',
+    'https://placekitten.com/g/450/450',
```

Comment:
As much as I love these, we should probably use something more _us_. `BpkImage` gets things from our CDN, can we do that here too? https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-image/stories.js#L29